lstchain/visualization/plot_dl2.py

```diff
@@ -132,44 +132,45 @@ def plot_features(data, truehadroness=False):
     plt.xlabel(r"Time gradient")
 
 
-def plot_e(data,truehadroness=False):
-    
-    """Plot the performance of reconstructed Energy. 
+def plot_e(data, Nbins, emin, emax, truehadroness=False):
```

Comment:
To be directly pep8 consistent, I would propose some changes:
`n_bins`, `true_hadroness`
`difE` --> be more explicit

Reply:
What about difE -> delta_e?